src/rosdep2/main.py

```diff
@@ -391,6 +399,18 @@ def _rosdep_main(args):
         check_for_sources_list_init(options.sources_cache_dir)
     elif command not in ['fix-permissions']:
         setup_proxy_opener()
+
+    if 'ROS_PYTHON_VERSION' not in os.environ and 'ROS_DISTRO' in os.environ:
+        # Set python version to version used by ROS distro
+        python_versions = MetaDatabase().get('ROS_PYTHON_VERSION')
```

Comment:
Looks like `MetaDatabase().get('ROS_PYTHON_VERSION')` can return `None` here, it's causing my CI builds to fail after they upgraded to rosdep 0.16.2: https://github.com/ros-infrastructure/rosdep/issues/720

Reply:
Fix proposed in #721